src/dispatching/dialogue/get_chat_id.rs

```diff
@@ -0,0 +1,13 @@
+use crate::types::Message;
+
+/// Something that has a chat ID.
+pub trait GetChatId {
```

Comment:
Is this trait needed? It is possible to write `ctx.message.chat_id()`

Reply:
Please, look at the definition or `SessionDispatcher`.